pyproject.toml

```diff
@@ -11,7 +11,6 @@ directory = "coverage"
 exclude_lines = [
     # Have to re-enable the standard pragma"
     "pragma: no branch",
-    "pragma: nocover",
```

Comment:
Why did you remove this?

Reply:
There was two occurences of "pragma: nocover",

Reply:
Yes, with different spelling. But it's okay 👍🏿